src/components/Footer.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import { Todo } from '../types/Todo';
+import { Status } from '../types/Status';
+import { Filter } from './Filter';
+import { Errors } from '../types/Errors';
+
+type Props = {
+  onFilter: (value: Status) => void;
```

Comment:
```suggestion
  onFilterChange: (value: Status) => void;
```
Try to name your variables/functions/etc so everyone can understand